src/main.c

```diff
@@ -19,14 +18,13 @@ void sys_tick_handler(void)
 	if (!collision_detected()) {
 		update_ideal_speed();
 		update_encoder_readings();
-		motor_control();
+		update_distance_readings();
```

Comment:
You removed the motor control. :wink:

Reply:
Sorry, I just commented it now. I prefer to test sensors without motors and encoders for now.